### DISCUSSION TITLE
What sports do you like? 🥎

### DISCUSSION THREAD

**Discussion by Ak99-S091** [NAIST-SE/AttractingContributors#827] [2023 Attraction Contributions]
Could you please me what isn your favorite sports? So please add your's lists of favorite sports to my README.md.

Please come to my fork: 
this is URL
https://github.com/Ak99-S091/AttractingContributors/blob/main/README.md

Please help me！！！

**Comment by KKKKKXY** [comment]
I added my favorite sport into your list (githubid: KKKKKXY).

**Comment by Sanaa3131** [comment]
I couldn't merge your pull request because of a branch conflict, do you know how to solve it? BTW, I have already contributed to your file

**Comment by WEI44ZHEYUAN** [comment]
I added 
Please help me
https://github.com/WEI44ZHEYUAN/AttractingContributors_Songs

**Comment by shanpeng-m** [comment]
🙆

**Comment by Gloria109** [comment]
I love playing tennis and horseback riding
